packages/studioCMS/src/index.ts

```diff
@@ -1,3 +1,62 @@
-import studioCMS from './studioCMS';
+import type { StudioCMSOptions } from './schemas';
+import coreIntegration from './coreIntegration';
+import { studioCMSRobotsTXT as robotsTXT } from './integrations';
+export * from './plugintools';
 
-export default studioCMS;
+export type { StudioCMSOptions };
+
+/**
```

Comment:
Love this  big ol comment 😍

Reply:
i fixed all the exported typing BS.... lol  So now the main exports can be done nice and clean